test/msdelay.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Since we use delay in `pickFuncProcess.ts`, you should make a `delay.ts` file in the `util` folder, remove it from `pickFuncProcess.ts,` and then export the util file and use it in both your test and `pickFuncProcess`